src/rdiff-backup-delete

```diff
@@ -60,6 +64,71 @@ def _str(value):
         return value.decode('utf-8', errors='replace')
 
 
+# Check if zcat is available.
+GZIP = spawn.find_executable('gzip')
```

Comment:
It's an internal "constant", its name should start with underscore.

Reply:
fixed